google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -89,6 +90,7 @@ def __init__(self, config):
         super(Crawler, self).__init__()
         self.config = config
 
+    @tracing.trace()
```

Comment:
@red2k18  @ocervell  Most of the methods in this class is decorated manually.  Why not decorate on the Class once?

Reply:
@blueandgold goal is to decorate only the methods that we need to trace, otherwise runtime will increase for the background thread exporting the spans. I'm not sure about which specific methods should be traced, so I've put it on most methods here. Let me know if you have an idea of which ones should / should not be traced.